readthedocs/templates/profiles/private/delete_account.html

```diff
@@ -9,12 +10,31 @@
 {% block edit_content_header %} {% trans "Delete Account" %} {% endblock %}
 
 {% block edit_content %}
+
+  {% block delete-warning %}
+    <p>
+      <p>
+        <strong>All projects where you are the only owner will be deleted.</strong>
+        If you want to keep a project, add another owner or transfer ownership.
```

Comment:
I think we could add a subtitle here. What about something like this?

----

## Delete Account

All projects where you are the only owner will be deleted.
If you want to keep a project, add another owner or transfer ownership.

### Projects to be delete

The following projects and all their documentation will be deleted:

- Project A
- Project B

<FORM Here>

----

Reply:
:+1: on the heading, we can skip the paragraph below the heading which is redundant.